docs/components/radio-group.md

```diff
@@ -49,4 +49,45 @@ const App = () => (
   </SlRadioGroup>
 );
 ```
+
+### Using the required attribute
```

Comment:
It would be helpful to have a "reset" button here since once a radio is checked, it can't be unchecked.

Reply:
Good call!